src/PartEditor/PartEditor.ts

```diff
@@ -367,14 +367,18 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider, Part
 
   private _partEditors: PartEditor[] = [];
 
-  public static register(context: vscode.ExtensionContext): vscode.Disposable {
```

Comment:
From [#1167](https://github.com/Samsung/ONE-vscode/issues/1167#issue-1329445337),

The function is tidy as is, but I am trying to standardize the registration function as below. 😄 

> Point 1. introduce `public static register(context: vscode.ExtensionContext) : void` function
> Point 2. Register each modules with the static function call
> ```
> OneTreeDataProvider.register(context);
> ```